feature-libs/organization/administration/components/shared/item-active.directive.ts

```diff
@@ -15,7 +15,9 @@ import { BaseItem } from './organization.model';
 @Directive({
   selector: '[cxOrgItemActive]',
 })
-export class ItemActiveDirective<T = BaseItem> implements OnInit, OnDestroy {
+export class ItemActiveDirective<T extends BaseItem = BaseItem>
```

Comment:
Adding a type to the generic type looks like a potential (compile time) breaking change.

Reply:
We are not adding a type, we just make the generic type more accurate. The new Angular seems more strict for type checking.